models/pull.js

```diff
@@ -240,4 +241,22 @@ Pull.getFromDB = function(data, signatures, comments, commitStatus, labels) {
    return new Pull(pullData, signatures, comments, commitStatus, labels);
 };
 
+Pull.isPassing = function() {
+   var required_builds = config.repos.find(function(repo) {
+      if (repo.full_name === this.repo) {
+         return repo.required_builds;
```

Comment:
Should the find callback just return whether or not the repo.full_name matches the repo name?

Reply:
79eb6e6